### PR TITLE
[Fluent] Notification

### PR DESCRIPTION
**Pull request by kikostadinov**
Related to: https://github.com/telerik/kendo-themes-private/issues/3

### CODE REVIEW

File: packages/fluent/scss/core/functions/_index.scss [hunk 1]
```diff
@@ -1 +1,2 @@
 // @forward "misc";
```
Comment (Juveniel) [inline packages/fluent/scss/core/functions/_index.scss:1]: We could probably remove this import, as well as the empty `_colors` module

File: packages/fluent/scss/notification/_layout.scss [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    // Base
+    .k-notification-group {
+        max-height: 100%;
+        display: inline-flex;
+        flex-flow: column-reverse wrap;
+        position: fixed;
+    }
+
+    .k-notification-container {
+        margin-block: math.div($kendo-notification-padding-y, 2);
```
Comment (Juveniel) [inline packages/fluent/scss/notification/_layout.scss:19]: use `calc()` instead and remove the `sass:math` ref

File: packages/fluent/scss/notification/_theme.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    .k-notification-wrap {
+        >.k-i-close {
+            color: inherit;
+        }
+    }
+
+    .k-notification-group .k-notification {
+        box-shadow: var(--INTERNAL--kendo-notification-shadow, var(--kendo-notification-shadow, #{$kendo-notification-shadow}));
```
Comment (Juveniel) [inline packages/fluent/scss/notification/_theme.scss:21]: From the design file it seems that there is no shadow for the Fluent Notification. Could you please double check this ?

File: packages/fluent/scss/notification/_variables.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
+@use "../core/color-system" as *;
+@use "../core/_variables.scss" as *;
+
+// Generic styles
+$info: #CEE5FF;
+$success: #DFF6DD;
+$warning: #FFF4CE;
+$error: #FDE7E9;
```
Comment (Juveniel) [inline packages/fluent/scss/notification/_variables.scss:8]: Only use colors defined in the color system. I would suggest extending the [default `$kendo-theme-colors` map](https://github.com/telerik/kendo-themes/blob/develop/packages/fluent/scss/core/color-system/_theme.scss#L67).

File: packages/fluent/scss/notification/_variables.scss [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+$kendo-fluent-status-colors: (
```
Comment (Juveniel) [inline packages/fluent/scss/notification/_variables.scss:10]: Naming too generic, we can use `$kendo-notification-theme-colors` here.

File: packages/fluent/scss/notification/_variables.scss [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    "info": $info,
+    "success": $success,
+    "warning": $warning,
+    "error": $error,
+    ) !default;
+
+// Notification
+$kendo-notification-padding-x: map-get($kendo-spacing, 2) !default;
```
Comment (Juveniel) [inline packages/fluent/scss/notification/_variables.scss:18]: add spacing

```suggestion
$kendo-notification-padding-x: map-get( $kendo-spacing, 2 ) !default;
```

File: packages/fluent/scss/notification/_variables.scss [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+$kendo-notification-padding-y: map-get($kendo-spacing, 1 ) !default;
```
Comment (Juveniel) [inline packages/fluent/scss/notification/_variables.scss:19]: add spacing

```suggestion
$kendo-notification-padding-y: map-get( $kendo-spacing, 1 ) !default;
```

File: packages/fluent/scss/notification/_variables.scss [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+$kendo-notification-border-width: 1px !default;
+$kendo-notification-border-radius: 2px; // $kendo-border-radius-md  !default;
```
Comment (Juveniel) [inline packages/fluent/scss/notification/_variables.scss:21]: use the global variable

```suggestion
$kendo-notification-border-radius:  var( --kendo-border-radius-md )
```

File: packages/fluent/scss/notification/_variables.scss [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+
+$kendo-notification-shadow: null !default; //$popup-shadow  !default;
```
Comment (Juveniel) [inline packages/fluent/scss/notification/_variables.scss:23]: Remove variable if not needed ( check design )

File: packages/fluent/scss/notification/_variables.scss [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+$kendo-notification-font-family: $kendo-font-family  !default;
```
Comment (Juveniel) [inline packages/fluent/scss/notification/_variables.scss:25]:
```suggestion
$kendo-notification-font-family: var( --kendo-font-family, inherit) !default;
```

File: packages/fluent/scss/notification/_variables.scss [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+$kendo-notification-font-size: 14px; //$kendo-font-size-sm  !default;
```
Comment (Juveniel) [inline packages/fluent/scss/notification/_variables.scss:26]:
```suggestion
$kendo-notification-font-size: var( --kendo-font-size, inherit ) !default;
```

File: packages/fluent/scss/notification/_variables.scss [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+$kendo-notification-line-height: calc(20 / 14); //$kendo-line-height  !default;
```
Comment (Juveniel) [inline packages/fluent/scss/notification/_variables.scss:27]:
```suggestion
$kendo-notification-line-height: var( --kendo-line-height, normal ) !default;
```

File: packages/fluent/scss/notification/_variables.scss [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+
+$kendo-notification-icon-offset: 2px !default;
```
Comment (Juveniel) [inline packages/fluent/scss/notification/_variables.scss:29]: get value from `$kendo-spacing`

File: packages/fluent/scss/notification/_variables.scss [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+$kendo-notification-icon-spacing: 4px !default; //$kendo-icon-spacing  !default;
```
Comment (Juveniel) [inline packages/fluent/scss/notification/_variables.scss:30]: get value from $kendo-spacing

File: packages/fluent/scss/notification/index.scss [hunk 15]
```diff
@@ -0,0 +1,25 @@
+// Module meta
+$_kendo-module-meta: (
+    name: "notification",
+    dependencies: (
+        "icons"
```
Comment (Juveniel) [inline packages/fluent/scss/notification/index.scss:5]: add "popup" as a dependency for the component

**Review by Juveniel — CHANGES_REQUESTED** [review]

**Review by Juveniel — APPROVED** [review]

**Review by TeyaVes — APPROVED** [review]

**Pull request merged**